flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/writer/BigQueryBufferedWriter.java

```diff
@@ -88,35 +91,52 @@ public class BigQueryBufferedWriter<IN> extends BaseWriter<IN>
 
     // Number of rows appended by this writer to current stream.
     private long appendRequestRowCount;
+    // Define counter which counts the number of records buffered by BigQuery Write API before
+    // actually sending them to the table.
+    // For these records in the writer, append() is called but are still
+    // awaiting flushRows() before being available in BQ.
+    // This count is maintained since the previous checkpoint,
+    // as total buffered records serve no practical value to the users.
```

Comment:
Please remove `as total buffered records serve no practical value to the users.`

Reply:
Fixed.